src/main/java/alison/tool/Storage.java

```diff
@@ -0,0 +1,63 @@
+package alison.tool;
+
+import alison.exception.AlisonException;
+import alison.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+
+    private String filePath;
+
+    public Storage(String path) {
+        this.filePath = path;
+    }
+
+    /**
+     * This method load the data from the hard disk when Alison starts up.
+     * @return TaskList converted from text file.
+     * @throws AlisonException
+     */
+    public TaskList load() throws AlisonException {
+        TaskList tasks = new TaskList();
+        try {
+            File dataFile = new File(filePath);
+            Scanner sc = new Scanner(dataFile);
+
+            while (sc.hasNextLine()) {
+                Task saved = Parser.parseTask(sc.nextLine());
+                if (saved != null) {
+                    tasks.add(saved);
+                }
+            }
+            sc.close();
+        } catch (FileNotFoundException e) {
+            throw AlisonException.loadingException();
+        }
+        return tasks;
+    }
+
+    /**
+     * This methods update the tasks list in your file automatically whenever the task list changes.
+     * @param tasks tasklist store in the bot.
```

Comment:
I think the Java coding standard specifies a break between the method description and start of `@param`.

Reply:
Thank you for the information! I didn't know it before.